optimize/util/optimize-commons/src/main/resources/service-config.yaml

```diff
@@ -607,7 +607,7 @@ panelNotification:
 
 entity:
   # which users are authorized to create/edit/delete Optimize entities outside of a collection.
-  # Available options: 'all', 'superuser', 'none'
```

Comment:
wanna remove the superUserIds and superGroupIds as well while we're here? Also there's another mention in the env config `Telemetry can later be enabled/disabled in the UI by superusers` (although I guess the telemetry config will also be removed?)

Reply:
Yeah I did a search on github and I think config here can be removed already, have pushed another commit doing that.

A separate commit clears up the note about telemetry in the UI